### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by maZahaca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,131 +1,207 @@
-# RedParser Phantom
+# goose-parser
 
-## Installing and run
+This library allows to parse grids, rows, and simple nodes from the page.
```
Comment (jifeon) [inline README.md]: Parse what? where? using what?

File: README.md [hunk 2]
```diff
@@ -1,131 +1,207 @@
...
+    scope: '.open-button.done'
+}
+```
 
-The purpose of this rule - retrieving simple textual node value
+### Transformations
 
-**Fields:**
+#### Date
+Format date to specific view.
```
Comment (jifeon) [inline README.md]: you need to mention moment lib here

File: README.md [hunk 3]
```diff
@@ -1,131 +1,207 @@
...
+* *actions* [optional]  - see [Actions](#actions).
+* *transform* [optional] - see [Transformations](#transformations).
 
-#### DataSet value
+#### Collection rule
 
-The purpose of this rule - retrieving collection of collection
+The purpose of this rule - retrieving collection of nodes.
+
+**Example:**
```
Comment (jifeon) [inline README.md]: there must be examples of source HTML and parsed data

File: README.md [hunk 4]
```diff
@@ -1,131 +1,207 @@
...
-        type: 'paginate',
-        scope: 'div.pageable > div.pagination > div'
     }
-);
-```
\ No newline at end of file
+}).then(function(parsed) {
+    
+});
+```
```
Comment (jifeon) [inline README.md]: paste link to file with tests and explanation how to run them

File: README.md [hunk 5]
```diff
@@ -1,131 +1,207 @@
-# RedParser Phantom
+# goose-parser
 
-## Installing and run
+This library allows to parse grids, rows, and simple nodes from the page.
+Parser supports paginations via infinity scroll and pages.
+It offers such features as pre-parse [actions](#actions) and post-parse [transformations](#transformations).
```
Comment (jifeon) [inline README.md]: I think it's necessary to mention about declarative format of validation rules

**Comment by jifeon** [comment]
I think it would be great to create a page with 3 textareas. First for HTML, second for parsing rules, third for results. But it's blocked by #9

**Comment by maZahaca** [comment]
#6

**Pull request merged**